example/app/layout.tsx

```diff
@@ -1,9 +1,11 @@
 import type { Metadata } from "next";
 import { Inter } from "next/font/google";
-import "./globals.css";
-import "@rainbow-me/rainbowkit/styles.css";
 import "@scaffold-ui/components/styles.css";
+import "@rainbow-me/rainbowkit/styles.css";
+// should be last or use !important in styles
```

Comment:
this is not true already but we need to recommend adding their css last so it will have priority in case of the same css specificity of elements